### PR TITLE
document undocumented vendor CLI commands

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: content/vendor/cli/app create.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Create a kots app.
+
+### Usage
+```bash
+replicated app create NAME
+```
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `-h, --help`   |  |          help for the command |
```
Comment (samantha-replicated) [inline content/vendor/cli/app create.md:18]: help >Help

File: content/vendor/cli/app create.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+| `--token string` | |  The API token to use to access your app in the Vendor API (default uses `$REPLICATED_API_TOKEN` env variable) |
```
Comment (samantha-replicated) [inline content/vendor/cli/app create.md:19]: to use to > used

File: content/vendor/cli/app delete.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Delete a kots app.
+
+### Usage
+```bash
+replicated app delete NAME
+```
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `-h, --help`   |  |          help for the command |
```
Comment (samantha-replicated) [inline content/vendor/cli/app delete.md:18]: help >Help

File: content/vendor/cli/app delete.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+| `-f, --force`  |  | Skip confirmation prompt. There is no undo for this action.
```
Comment (samantha-replicated) [inline content/vendor/cli/app delete.md:19]: Skip the confirmation prompt. This action cannot be undone.

File: content/vendor/cli/app delete.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+| `--token string` | |  The API token to use to access your app in the Vendor API (default uses `$REPLICATED_API_TOKEN` env variable) |
```
Comment (samantha-replicated) [inline content/vendor/cli/app delete.md:20]: to use to > used

File: content/vendor/cli/app delete.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+### Examples
+```bash
+replicated app delete laverya-deletion-example
+  • Fetching App ✓
+ID                             NAME                        SLUG                        SCHEDULER
+1xyAIzrmbvqHzzcKPbTmmrTxhl4    laverya-deletion-example    laverya-deletion-example    kots
```
Comment (samantha-replicated) [inline content/vendor/cli/app delete.md:27]: rm laverya- in all code blocks

File: content/vendor/cli/app ls.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+|:----------------------|------|-------------|
+| `-h, --help`   |  |          help for the command |
+| `--token string` | |  The API token to use to access your app in the Vendor API (default uses `$REPLICATED_API_TOKEN` env variable) |
+
+### Examples
+```bash
+replicated app ls
+ID                                  NAME                                  SLUG                              SCHEDULER
+cqzmRRIdJtZ_E82Cl36nkBDSmGrG2uoe    sentry-enterprise                     sentry-enterprise-1               kots
+1xy9t8G9CO0PRGzTwSwWFkMUjZO         laverya-cli-app                       laverya-cli-app                   kots
```
Comment (samantha-replicated) [inline content/vendor/cli/app ls.md:26]: rm laverya- in all code blocks

File: content/vendor/cli/channel inspect.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+replicated channel inspect cli-created
+ID:             1xyB2Mgbg9N7rExShfbdBYIuzeW
+NAME:           cli-created
+DESCRIPTION:
+RELEASE:        1
+VERSION:        0.0.1
+EXISTING:
+
+    curl -fsSL https://kots.io/install | bash
+    kubectl kots install laverya-cli-app/cli-created
```
Comment (samantha-replicated) [inline content/vendor/cli/channel inspect.md:33]: rm laverya- in all code blocks

File: content/vendor/cli/app ls.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
+---
+date: 2021-09-10
+linktitle: "app ls"
+title: "app ls"
+weight: 90105
+draft: false
+---
+
+List all created kots apps, or one in particular.
```
Comment (samantha-replicated) [inline content/vendor/cli/app ls.md:9]: List all created kots apps, or specify the kots app you want to list.

File: content/vendor/cli/app ls.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+### Usage
+```bash
+replicated app ls
+```
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `-h, --help`   |  |          help for the command |
```
Comment (samantha-replicated) [inline content/vendor/cli/app ls.md:18]: help >Help

File: content/vendor/cli/app ls.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+| `--token string` | |  The API token to use to access your app in the Vendor API (default uses `$REPLICATED_API_TOKEN` env variable) |
```
Comment (samantha-replicated) [inline content/vendor/cli/app ls.md:19]: to use to > used

File: content/vendor/cli/app ls.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+### Examples
+```bash
```
Comment (samantha-replicated) [inline content/vendor/cli/app ls.md:22]: To split code blocks add the below line text before the block...

List all created kots apps:

File: content/vendor/cli/app ls.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+replicated app ls
+ID                                  NAME                                  SLUG                              SCHEDULER
+cqzmRRIdJtZ_E82Cl36nkBDSmGrG2uoe    sentry-enterprise                     sentry-enterprise-1               kots
+1xy9t8G9CO0PRGzTwSwWFkMUjZO         laverya-cli-app                       laverya-cli-app                   kots
+XlkTz6qxXepG_HjK_vUFlLUY14p05wO1    laverya-default-kots                  laverya-default-kots              kots
+```
+
```
Comment (samantha-replicated) [inline content/vendor/cli/app ls.md:29]: To split code blocks add the below line text before the block...

List a specific kots app:

File: content/vendor/cli/channel create.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
+---
+date: 2021-09-10
+linktitle: "channel create"
+title: "channel create"
+weight: 90110
+draft: false
+---
+
+Create a new channel in your app, and print the channel on success.
```
Comment (samantha-replicated) [inline content/vendor/cli/channel create.md:9]: Create a new channel in your app, and then print the channel.

File: content/vendor/cli/channel create.md [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+### Usage
+```bash
+replicated channel create --name Beta --description 'New features subject to change'
+```
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `--description` | string | A longer description of this channel |
+| `--name` | string | The name of this channel (**required**) |
+| `-h, --help`   |  |          help for the command |
```
Comment (samantha-replicated) [inline content/vendor/cli/channel create.md:20]: help >Help

File: content/vendor/cli/channel create.md [hunk 16]
```diff
@@ -0,0 +1,29 @@
...
+| `--app string` | |   The app slug or app id to use in all calls (default uses `$REPLICATED_APP` env variable) |
```
Comment (samantha-replicated) [inline content/vendor/cli/channel create.md:21]: to use > used

File: content/vendor/cli/channel create.md [hunk 17]
```diff
@@ -0,0 +1,29 @@
...
+| `--token string` | |  The API token to use to access your app in the Vendor API (default uses `$REPLICATED_API_TOKEN` env variable) |
```
Comment (samantha-replicated) [inline content/vendor/cli/channel create.md:22]: to use to > used

File: content/vendor/cli/channel delete.md [hunk 18]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+Delete (archive) a channel.
+
+### Usage
+```bash
+replicated channel rm CHANNEL_ID
+```
+
+Only the channel ID will work, not the name.
```
Comment (samantha-replicated) [inline content/vendor/cli/channel delete.md:16]: **Note:** You can only use the channel ID to delete the channel, not the channel name.

File: content/vendor/cli/channel delete.md [hunk 19]
```diff
@@ -0,0 +1,28 @@
...
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `-h, --help`   |  |          help for the command |
```
Comment (samantha-replicated) [inline content/vendor/cli/channel delete.md:20]: help > Help

File: content/vendor/cli/channel delete.md [hunk 20]
```diff
@@ -0,0 +1,28 @@
...
+| `--app string` | |   The app slug or app id to use in all calls (default uses `$REPLICATED_APP` env variable) |
```
Comment (samantha-replicated) [inline content/vendor/cli/channel delete.md:21]: to use > used

File: content/vendor/cli/channel delete.md [hunk 21]
```diff
@@ -0,0 +1,28 @@
...
+| `--token string` | |  The API token to use to access your app in the Vendor API (default uses `$REPLICATED_API_TOKEN` env variable) |
```
Comment (samantha-replicated) [inline content/vendor/cli/channel delete.md:22]: to use to >used

File: content/vendor/cli/channel inspect.md [hunk 22]
```diff
@@ -0,0 +1,55 @@
...
+Show the full details for a channel.
+
+### Usage
+```bash
+replicated channel inspect CHANNEL_ID
+```
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `-h, --help`   |  |          help for the command |
```
Comment (samantha-replicated) [inline content/vendor/cli/channel inspect.md:18]: help > Help

to use > used

to use to > used

File: content/vendor/cli/customer download-license.md [hunk 23]
```diff
@@ -0,0 +1,46 @@
+---
+date: 2021-09-10
+linktitle: "customer download-license"
+title: "customer download-license"
+weight: 90117
+draft: false
+---
+
+Print the license YAML for a customer
```
Comment (samantha-replicated) [inline content/vendor/cli/customer download-license.md:9]: add period

File: content/vendor/cli/customer download-license.md [hunk 24]
```diff
@@ -0,0 +1,46 @@
...
+
+### Usage
+```bash
+replicated customer download-license
+```
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `--customer` | string | The Customer Name or ID |
+| `-h, --help`   |  |          help for the command |
```
Comment (samantha-replicated) [inline content/vendor/cli/customer download-license.md:19]: help > Help

to use > used

to use to > used

File: content/vendor/cli/customer download-license.md [hunk 25]
```diff
@@ -0,0 +1,46 @@
...
+
+### Examples
+```bash
+replicated customer download-license --customer cli-customer
+apiVersion: kots.io/v1beta1
+kind: License
+metadata:
+  name: cli-customer
+spec:
+  appSlug: laverya-cli-app
```
Comment (samantha-replicated) [inline content/vendor/cli/customer download-license.md:31]: rm laverya- from all code blocks

File: content/vendor/cli/installer create.md [hunk 26]
```diff
@@ -0,0 +1,67 @@
+---
+date: 2021-09-10
+linktitle: "installer create"
+title: "installer create"
+weight: 90115
+---
+
+Create a new installer spec by providing YAML configuration for a https://kurl.sh cluster.
```
Comment (samantha-replicated) [inline content/vendor/cli/installer create.md:8]: Create a new installer spec by providing a YAML configuration file for a https://kurl.sh cluster.

File: content/vendor/cli/installer create.md [hunk 27]
```diff
@@ -0,0 +1,67 @@
...
+
+### Usage
+```bash
+replicated installer create [flags]
+```
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+|  `--auto`            | |    generate default values for use in CI
```
Comment (samantha-replicated) [inline content/vendor/cli/installer create.md:17]: Generate the default values for use in the CI

File: content/vendor/cli/installer create.md [hunk 28]
```diff
@@ -0,0 +1,67 @@
...
+| `-y, --confirm-auto` | |       auto-accept the configuration generated by the --auto flag
```
Comment (samantha-replicated) [inline content/vendor/cli/installer create.md:18]: auto >Auto

File: content/vendor/cli/installer create.md [hunk 29]
```diff
@@ -0,0 +1,67 @@
...
+|  `--ensure-channel`   | |  When used with --promote <channel>, will create the channel if it doesn't exist
```
Comment (samantha-replicated) [inline content/vendor/cli/installer create.md:19]: When used with --promote <channel>, this command creates the channel if it doesn't already exist

File: content/vendor/cli/installer create.md [hunk 30]
```diff
@@ -0,0 +1,67 @@
...
+|  `--promote` | string | Channel name or id to promote this installer to
+|  `--yaml` | string |        The YAML config for this installer. Use '-' to read from stdin. Cannot be used with the --yaml-file flag.
+|  `--yaml-file` | string |  The file name with YAML config for this installer. Cannot be used with the --yaml flag.
+| `-h, --help`   |  |          help for admin-console |
```
Comment (samantha-replicated) [inline content/vendor/cli/installer create.md:23]: Help for the admin-console

File: content/vendor/cli/installer create.md [hunk 31]
```diff
@@ -0,0 +1,67 @@
...
+| `--app string` | |   The app slug or app id to use in all calls (default uses `$REPLICATED_APP` env variable) |
```
Comment (samantha-replicated) [inline content/vendor/cli/installer create.md:24]: to use > used

to use to >used

File: content/vendor/cli/installer ls.md [hunk 32]
```diff
@@ -0,0 +1,29 @@
+---
+date: 2021-09-10
+linktitle: "installer ls"
+title: "installer ls"
+weight: 90115
+---
+
+List an app's https://kurl.sh Kubernetes Installers
```
Comment (samantha-replicated) [inline content/vendor/cli/installer ls.md:8]: List the https://kurl.sh Kubernetes Installers for an app.

File: content/vendor/cli/release download.md [hunk 33]
```diff
@@ -0,0 +1,37 @@
+---
+date: 2021-09-10
+linktitle: "release download"
+title: "release download"
+weight: 90121
+---
+
+Download the YAML config for a release. Same as 'release inspect' for non-KOTS apps.
```
Comment (samantha-replicated) [inline content/vendor/cli/release download.md:8]: Download the YAML config file for a release.

**Note:** This command is the same as the `release inspect` command for non-KOTS apps.

File: content/vendor/cli/release download.md [hunk 34]
```diff
@@ -0,0 +1,37 @@
...
+
+### Usage
+```bash
+replicated release download SEQUENCE -d ./appyaml
+```
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `-d, --dest` | string  | Directory to which release manifests should be downloaded |
```
Comment (samantha-replicated) [inline content/vendor/cli/release download.md:17]: Help for the admin-console

to use > used

to use to >used

**Review by samantha-replicated — CHANGES_REQUESTED** [review]
I tried to make it easy and pull a commit against your PR...but git wouldn't let me. So here are the some minor edit suggestions. Let me know if you have any questions :)

File: content/vendor/cli/app create.md (hunk 2 above)
Comment (laverya) [reply content/vendor/cli/app create.md:19]: to be clear, this was duplicated from what some existing docs said
I'll change it for all of them

**Review by samantha-replicated — APPROVED** [review]
LGTM! Thanks for incorporating those edits.

**Pull request merged**